src/views/Pipeline.vue

```diff
@@ -180,7 +180,7 @@
           </div>
 
           <div v-else>
-          <ion-card v-for="job in jobHistory" :key="job.jobId">
+          <ion-card v-for="job in jobHistory" :key="job.jobId" @click="viewJobConfiguration(job)">
```

Comment:
We have some click events defined in child DOM of the ion-card, just cross check that those will not collide with this once.